sdk/core/Azure.Core/src/AccessToken.cs

```diff
@@ -49,12 +66,17 @@ public AccessToken(string accessToken, DateTimeOffset expiresOn, DateTimeOffset?
         /// </summary>
         public DateTimeOffset? RefreshOn { get; }
 
+        /// <summary>
+        /// Identifies the type of access token.
+        /// </summary>
+        public string TokenType { get; }
```

Comment:
Is this just meant to be informational?  If so, we may want to add that to the remarks here to add guidance for "don't take a hard dependency on this value in logic" kinda thing.   If not, should we make this an enum?

Reply:
it's informational, but gets populated by MSAL from their token results - it defaults to `Bearer` if no value is supplied. I suppose it could be an enum but it's just a string in MSAL, and this seems the most compatible.

Reply:
Yes, but the values are fairly well known. Curious if @KrzysztofCwalina has any thoughts here on whether we should make this more strongly typed.

Reply:
extensible enum wont help in hiding/abstracting the value (so users don't take a dependency on it). We could create a wrapper typer, but I don't have a good feel for whether it would be worth or not; I am not sure how likely it is that users would try to take a dependency on the exact value.

Reply:
discussed offline - since this is generally an output only property an extensible enum would be of limited value here.